contents/blog/pricing-lessons.md

```diff
@@ -0,0 +1,96 @@
+---
+date: 2022-03-17
+title: The pricing guide I never had
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+---
+
+Imagine being able to grow twice as fast with a few hours of work. Changing the way you price has a real chance to get you there. 
+
+Here is a breakdown of the counterintuitive lesson I've learned as a cofounder at PostHog - the more we give away for free, and the less we ask from users on day 1, the _more_ money we make.
+
+## Don't let pricing get in the way of adoption
+
+Most software companies agonize over how their product is adopted yet when it comes to pricing, they default to charging a certain recurring amount per month. The only thing under deep consideration is how _much_... not _how_.
+
+Our revelation was realizing that, for us, pricing shouldn't get in the way of adoption, because we don't sell top down into companies.. our most successful paying customers get there as follows:
```

Comment:
```suggestion
We realized that our pricing shouldn't get in the way of adoption, because we don't sell top-down into companies. Instead, our most successful paying customers get there as follows:
```
I think this is just simpler construction